summaries/consts.py

```diff
@@ -22,4 +22,17 @@
     (SUMMARY_TYPE_KSG_IT, 'KSG-IT summary')
 )
 
+SUMMARY_TYPE_SHORT_NAMES = (
+    (SUMMARY_TYPE_OTHER, 'Other'),
+    (SUMMARY_TYPE_KAFE_ANSVARLIG, 'Kafeansvarlig'),
+    (SUMMARY_TYPE_HOVMESTER, 'Hovmester'),
+    (SUMMARY_TYPE_BARSJEF, 'Barsjef'),
+    (SUMMARY_TYPE_SOUSCHEF, 'Souschef'),
+    (SUMMARY_TYPE_ARRANGEMENT, 'Arrangement'),
+    (SUMMARY_TYPE_OKONOMI, 'Økonomi'),
+    (SUMMARY_TYPE_STYRET, 'Styret'),
+    (SUMMARY_TYPE_KSG_IT, 'KSG-IT')
+)
```

Comment:
Missing Driftsmøtet and DH?

Reply:
Yeah, you're right, thanks! This seems to be generally missing, so I'll create a new branch.